cmd/admission-webhook/main.go

```diff
@@ -61,7 +60,14 @@ func main() {
 		return
 	}
 
-	logger, err := logging.NewLogger(logConfig)
+	logger, err := logging.NewLoggerSlog(logConfig)
+	if err != nil {
+		stdlog.Fatal(err)
+	}
+
+	// The old logger will be removed in the future. For now, we are using it to avoid breaking changes and
+	// to have a smooth transition.
+	oldLogger, err := logging.NewLogger(logConfig)
```

Comment:
Does this approach make sense? I think that will be easier to review and change the dependencies code if we do that incrementally.